doc/PluginTutorial.md

```diff
@@ -164,7 +164,7 @@ section:
 ```html
 	<head>
 		<!-- CSS -->
-		<link rel="stylesheet" href="../../css/roboto-condensed-min.css">
+		<link rel="stylesheet" href="../../css/roboto-condensed/css/roboto-condensed-min.css">
```

Comment:
I agree with Ming, try to keep the directory structure as simple as possible.

Some of the node.js stuff is in love with layered modules and whatnot, but it can make it difficult to find things and often causes problems in Windows where filepaths are only allowed to be 255 characters (starting from C:\) in the first place.